PlanningTools.xaml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Lab6_Starter.PlanningTools"
+             Title="Planning Tools">
+        <Grid RowDefinitions="*,2*,25,2*,*"
+              ColumnDefinitions="*,3*,25,3*,*">
+                <Button Grid.Row="1"
```

Comment:
There's a problem on iOS, the button text is being cutoff. This is one of those pain points of doing cross-platform, and .NET MAUI ought to handle it but does not. So, make the Buttons wider - maybe 4* instead of 3*, and try throwing this in platform-specific font-sizes as well:

<Button Grid.Row="1"
        Grid.Column="1"
        Text="Nearby Airports"
        FontSize="{OnPlatform iOS=14, Android=16}". <!-- add this -->
   HorizontalOptions="FillAndExpand"
        VerticalOptions="FillAndExpand" />

This might not solve the problem, but it'll be a start. If you want to drop by during office hours, we can play with these settings to get them just right.